.github/workflows/code_style.yml

```diff
@@ -26,9 +20,18 @@ jobs:
             clang-format \
             git
 
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch ${{ github.base_ref }}
+        shell: bash
+        run: |
+          git fetch --progress --depth=1 origin ${{ github.base_ref }}
+          git log --pretty=oneline
+
       - name: Style check
         shell: bash
         run: |
-          git-clang-format origin/master
+          git-clang-format origin/${{ github.base_ref }}
```

Comment:
I can't believe I didn't notice that `${{ github.base_ref }}` exists when writing this lol. Also, I'm not sure if the fetch step after the checkout is really required?

Reply:
It does not know otherwise about the base_ref branch, only the head/ source branch.